### PR TITLE
[swift5] update visibility modifiers

### PR DESCRIPTION
**Pull request by 4brunu**
There is a delegate method that is public, but it shouldn't be, this PR fixes that.

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [x] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

@jgavris (2017/07) @ehyche (2017/08) @Edubits (2017/09) @jaz-ah (2017/09) @4brunu (2019/11)

### CODE REVIEW

File: samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Test serialization of outer boolean types
 // The following code samples are still beta. For any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
 import PetstoreClient
 
-let body = false // Bool | Input boolean as post body (optional)
+let body = true // Bool | Input boolean as post body (optional)
```
Comment (wing328) [inline samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md]: @4brunu I notice that the value has been flipped from `true` to `false` int the sample code. Is that expected?

Comment (4brunu) [reply samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md]: I was also surprised.
That happen after running the pet projects. 
The input spec has changed, but I didn't change it.

Comment (wing328) [reply samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md]: 👌

Comment (wing328) [reply samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md]: I'll take another look after merging this PR instead.

Comment (wing328) [reply samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md]: Looks like that's the correct behavior: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L1421 (and it was fixed somehow)

**Pull request merged**